e2e/README.md

```diff
@@ -115,20 +127,20 @@ is greater than or equal to 1.17.0.
     should be `v1beta1`
 
     ```console
-    $./install-snapshot.sh delete-crd
+    $./scripts/install-snapshot.sh delete-crd
```

Comment:
missing space after `$` is not too bad here, but see below

Reply:
I have this style too. It looks good when we have one space after $, but the CI hates it.

```
./e2e/README.md:129: MD014 Dollar signs used before commands without showing output
./e2e/README.md:155: MD014 Dollar signs used before commands without showing output
./e2e/README.md:161: MD014 Dollar signs used before commands without showing output
```